spdx-utils/src/main/kotlin/SpdxLicense.kt

```diff
@@ -853,12 +853,12 @@ enum class SpdxLicense(
 
                 val isGpl = listOf("AGPL-", "GPL-", "LGPL-").any { baseId.startsWith(it) }
                 if (isGpl) {
-                    // Note: Do not use appendln() here as that would write out platform-native line endings, but we
-                    // want to normalize on Unix-style line endings for consistency.
+                    // Note: Do not use appendln() here as that would write out platform-native line endings, but
```

Comment:
Should this commit, according to its commit message, just do change in the enums?

Reply:
That's the formatting change from two commits earlier applied to the generated classes.